### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by LyWangPX** [rosinality/glow-pytorch#20]
Paper indicates learning rate be 1e-3 in Appendix C. I wonder why the default of the code is set to be 1e-4. 
Any empirical reasons for you to do that?

Update: I found the description in the README.
